src/main/webapp/app/entities/theory/theory.component.ts

```diff
@@ -1,22 +1,33 @@
+import { ActivatedRoute } from "@angular/router";
 import { Component, OnInit, OnDestroy } from '@angular/core';
 import { HttpErrorResponse, HttpHeaders, HttpResponse } from '@angular/common/http';
+import {Location, LocationStrategy, PathLocationStrategy} from '@angular/common';
```

Comment:
Really? Have you seen another formatting? Open your eyes and look into the row above!